src/test/java/hec/stats/MeanComputableTest.java

```diff
@@ -8,8 +8,15 @@
     @Test
     public void testMeanCompute() {
         MeanComputable test = new MeanComputable();
-        float[] num = {1,2,3,4,5,6};
+        float[] num = {1,2,3,4,5,6,7,8};
         float results = test.compute(num);
-        assertEquals(3.5, results);
+        assertEquals(4.5, results);
+    }
+
+    {
```

Comment:
It really looks like this test was not ever run - it seems this code would not compile properly. You have no method signature. You would need to name this method something like testMeanCompute_NonSequential() and annotate it as "@test"